src/tty/unix.rs

```diff
@@ -185,9 +205,11 @@ fn get_pw_entry(buf: &mut [i8; 1024]) -> Passwd<'_> {
 }
 
 pub struct Pty {
+    child: Child,
     pub fd: File,
-    pub raw_fd: RawFd,
```

Comment:
I took the liberty of cleaning this up - why keep this when you can `fd.as_raw_fd()`?